docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md

```diff
@@ -0,0 +1,136 @@
+# Use Cascading Inputs in Tool
+
+Cascading settings between inputs are frequently used in situations where a selection in one input field determines what subsequent inputs should be shown. This approach helps create a more efficient, user-friendly, and error-free input process. This article will guide you through the process of implementing cascading settings for tool inputs.
+
+## Prerequisites
+Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.2.0 or later.
```

Comment:
Please make sure you have the latest version of [Prompt Flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) installed (v1.2.0+).